Document/0x04g-Testing-Cryptography.md

```diff
@@ -130,6 +134,14 @@ Through source code analysis the following non-exhausting configuration options
 - Fit-for-purpose block encryption modes
 - Key management being done properly
 
+#### Dynamic Analysis
+
+Various weaknesses can be detected within out access to the source code:
+
+- if access to encrypted files or to generated hashes is possible (e.g., through stored files) the same file can be encrypted by two users. If this results in the same encrypted file, a static encryption key has been used.
```

Comment:
I think this sentence could be made a bit clearer. I was confused by the first sentence:

> if access to encrypted files or to generated hashes is possible (e.g., through stored files) the same file can be encrypted by two users.

You have here an "if a, then b", but b is not dependant on a. The same file can be encrypted by two users whether access to the encrypted files is possible or not. Furthermore, this does not indicate a weakness, which only comes in the next sentence. I would change it to something like this:

> Let two users encrypt a file. If this results in the same encrypted file, the app uses a static encryption key. Access to the encrypted file is necessary to confirm this weakness.